ccmlib/node.py

```diff
@@ -1254,22 +1257,34 @@ def _update_pid(self, process):
             raise NodeError('Problem starting node %s' % self.name, process)
         self.__update_status()
 
-    def __gather_sstables(self, datafile=None, keyspace=None, columnfamilies=None):
-        datafiles = []
+    def __gather_sstables(self, datafiles=None, keyspace=None, columnfamilies=None):
+        files = []
         if keyspace is None:
-            for k in self.list_keyspaces():
-                datafiles = datafiles + self.get_sstables(k, "")
-        elif datafile is None:
+            if datafiles:
+                for datafile in datafiles:
+                    if not os.path.isabs(datafile):
+                        datafile = os.path.join(os.getcwd(), datafile)
```

Comment:
Changes proposed:

1. To use datafiles, makes Keyspace param (-k) mandatory 
2. To use datafiles, makes CF param (-c) mandatory 
3. Since I want to support absolute and relative paths matching though bash... an exception will be thrown if the one of the datafiles specified is NOT in the keyspace + column family directory specified in command line.

Specifying a datafile without keyspace and cf will result in the same behavior than before.